src/stream/src/executor/mview/materialize.rs

```diff
@@ -119,7 +119,10 @@ impl<S: StateStore> MaterializeExecutor<S> {
                         let row = Row(chunk
                             .columns()
                             .iter()
-                            .map(|x| x.array_ref().datum_at(idx))
+                            .enumerate()
+                            // TODO: optimize + handle memcomparable != value encoding
+                            .filter(|(col_idx, _)| !self.arrange_columns.contains(col_idx))
+                            .map(|(_, col)| col.array_ref().datum_at(idx))
```

Comment:
Seems like chunk dimensions were affected:
```
thread 'thread 'tokio-runtime-workertokio-runtime-worker' panicked at '' panicked at 'assertion failed: `(left == right)`
  left: `3`,
 right: `4`assertion failed: `(left == right)`
  left: `3`,
 right: `4`', ', src/common/src/util/ordered/serde.rs:src/common/src/util/ordered/serde.rs166::1665
```
Need to modify chunk decode logic (where?).